clang/test/CodeGenOpenCL/check-atomic-alignment.cl

```diff
@@ -0,0 +1,45 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+// RUN: %clang_cc1 -O0 -cl-std=CL2.0 -triple amdgcn-amd-amdhsa -target-cpu gfx942 \
+// RUN:   %s -emit-llvm -o - | FileCheck %s
+
+// REQUIRES: amdgpu-registered-target
```

Comment:
It shouldn't actually

Reply:
More or less yes. And even then it doesn't matter most of the time, it just skips the backend registered passes (plus TTI doesn't work)

Reply:
but I guess we have this on most of the tests, so might as well keep it